pages/projects/[projectId]/collections/[collectionId]/edit.tsx

```diff
@@ -91,21 +93,41 @@ export default function EditPage(props: Props) {
                   <div className="px-4 py-5 bg-white space-y-6 sm:p-6">
                     <div>
                       <label
-                        htmlFor="name"
+                        htmlFor="collectionName"
                         className="block text-sm font-medium text-gray-700"
                       >
                         Collection Name
                       </label>
                       <input
                         type="text"
-                        name="name"
-                        id="name"
+                        name="collectionName"
+                        id="collectionName"
                         placeholder="Gen 0"
-                        defaultValue={collection.name}
+                        defaultValue={collection.collectionName}
                         className="mt-1 block w-full shadow-sm sm:text-sm rounded-md"
                       />
                     </div>
 
+                    <div className="col-span-6 sm:col-span-4">
+                      <label
+                          htmlFor="name"
+                          className="block text-sm font-medium text-gray-700"
+                      >
+                        NFT Name
+                      </label>
+                      <input
+                          type="text"
+                          name="name"
+                          id="name"
+                          placeholder="Name of NFT"
+                          defaultValue={collection.name}
+                          className="mt-1 block w-full shadow-sm sm:text-sm rounded-md"
+                      />
+                      <p className="mt-2 text-xs text-gray-500">
+                        This is the name of the NFT that will be generated in the JSON file
```

Comment:
can we note in this description that the name will have the index number appended?

"This is the name that will be shown on the NFT. The index number will be appended, i.e. "NAME 542".

Reply:
Will do tomorrow!